src/components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,167 @@
+@import '../Media/Media.scss';
+
+.root {
+  position: relative;
+
+  &.block {
+    margin-left: 5px;
+    display: block;
+  }
+
+  &.inline {
+    display: inline-block;
+    vertical-align: middle;
+  }
+}
+
+/* Tooltip */
+.icon {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.popperContainer {
+  opacity: 0;
+  animation: fadein 0.3s ease-in 0.5s forwards;
+
+  .contentBox {
+    @include for-phone-only {
+      display: none;
+    }
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 12px;
+    text-align: center;
+    width: max-content;
+    max-width: 270px;
+    background-color: white;
+    font-size: 12px;
+    padding: 20px var(--Space-24) var(--Space-24) var(--Space-24);
+    box-shadow: 0px 0px 12px 5px #f1f1f1f1;
+    color: black;
+    transition: 0.1s ease-in-out;
+
+    &.visible {
+      opacity: 1;
+      visibility: visible;
+    }
+
+    &.hidden {
+      opacity: 0;
+      visibility: hidden;
+    }
+  }
+
+  .arrow {
+    position: absolute;
+    width: 12px;
+    height: 12px;
+    &[data-placement*='bottom'] {
+      top: 0;
+      margin-top: -12px;
+      width: 36px;
+      height: 12px;
+      &::before {
+        border-width: 0 12px 12px 12px;
+        border-color: transparent transparent white transparent;
+      }
+    }
+    &[data-placement*='top'] {
+      bottom: 0;
+      margin-bottom: -12px;
+      width: 36px;
+      height: 12px;
+      &::before {
+        border-width: 12px 12px 0 12px;
+        border-color: white transparent transparent transparent;
+      }
+    }
+    &[data-placement*='right'] {
+      left: 0;
+      margin-left: -12px;
+      height: var(--Space-24);
+      width: 12px;
+      &::before {
+        border-width: 12px 12px 12px 0;
+        border-color: transparent white transparent transparent;
+      }
+    }
+    &[data-placement*='left'] {
+      right: 0;
+      margin-right: -12px;
+      height: var(--Space-24);
+      width: 12px;
+      &::before {
+        border-width: 12px 0 12px 12px;
+        border-color: transparent transparent transparent white;
+      }
+    }
+    &::before {
+      content: '';
+      margin: auto;
+      display: block;
+      width: 0;
+      height: 0;
+      border-style: solid;
+    }
+  }
+}
+
+@keyframes fadein {
```

Comment:
Can we centralize all keyframes for reuse?

Reply:
Remove keyframes